CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Updated all dashboars using `decbytes` unit to use `bytes` (IEC units) instead.
+- alertmanager dashboard: show related logs
```

Comment:
```suggestion
- Update alertmanager dashboard to show related logs
```

Reply:
I like the synthetic "component: change" syntax.
I'd rather align to your other comment with something like:
```
- alertmanager dashboard: added a logs panel
```

Reply:
We do use descriptive entries in our changelogs across the organization, so let's stick with that.

Reply:
All right.